config/config.go

```diff
@@ -376,17 +377,31 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 			}
 
 			if wcc.APISecret == "" {
-				if c.Global.WeChatAPISecret == "" {
-					return fmt.Errorf("no global Wechat ApiSecret set")
+				// Use Environment Variable First
+				apiSecret := Secret(os.Getenv("APISecret"))
```

Comment:
We don't support configuration by environment variables. It's an organization-wide policy (see https://github.com/prometheus/prometheus/issues/2357) so not specific to AlertManager.

Reply:
I want to protect the apisecret, so use environment variables to store secrets。
I will fix it.

Reply:
> IIUC the user can't configure `ToAppChat` and any of `ToUser`, `ToParty` or `Totag` at the same time. If correct, it should be checked when loading the configuration. The same goes for `MessageType` which can only have 2 values from what I understand.
> 
> Do we want default values for the new attributes?
> 
> Is there an English version of the API documentation?

1.ToAppChat ToUser, ToParty or Totag can config at the same time, 
2.if ToAppChat is not empty, and ToUser/ToParty/Totag make no sense.
3. MessageType have two value "text" and "textcard", it Represents the representation of the message content, I will add default value "text". thanks

Reply:
> if ToAppChat is not empty, and ToUser/ToParty/Totag make no sense.

In general, we prefer to prevent such cases as it could confuse users. I suggest that you expand the `(c *WechatConfig) UnmarshalYAML` method so it fails when someone configures ToAppChat with ` ToUser`, `ToParty` or `Totag`.

> MessageType have two value "text" and "textcard", it Represents the representation of the message content, I will add default value "text". thanks

Same here, if `MessageType` is something else than the expected values then it should fail.